src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,132 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int capasity;
```

Comment:
you can get capacity using `table.length` you don't need an additional class-level variable for this.

Reply:
but, when i do resize i need calculate new index and if i use table.length i will have problem, because it length will be old and i need new length. Example : i need new index wiht length 32, but program will calculate index with length 16